system/jlib/jtrace.cpp

```diff
@@ -655,13 +655,23 @@ class CServerSpan : public CSpan
         return success;
     }
 
+    void setContextAttributes()
+    {
+        if (!isEmptyString(hpccGlobalId))
+            setSpanAttribute(kGlobalIdHttpHeaderName, hpccGlobalId.get());
```

Comment:
For discussion: Is this the correct name to use for the attribute?  Would "hpcc.globalid" be more appropriate?
(See https://opentelemetry.io/docs/specs/otel/common/attribute-naming/)
Possible something to discuss more broadly before we merge.